content/zh/docs/concepts/architecture/nodes.md

```diff
@@ -190,82 +388,25 @@ This information is gathered by Kubelet from the node.
 
 ### 信息 {#info}
 
-关于节点的通用信息，例如内核版本、Kubernetes 版本（kubelet 和 kube-proxy 版本）、Docker 版本
-（如果使用了）和操作系统名称。这些信息由 kubelet 从节点上搜集而来。
-
-<!--
-## Management
-
-Unlike [pods](/docs/concepts/workloads/pods/pod/) and [services](/docs/concepts/services-networking/service/),
-a node is not inherently created by Kubernetes: it is created externally by cloud
-providers like Google Compute Engine, or it exists in your pool of physical or virtual
-machines. So when Kubernetes creates a node, it creates
-an object that represents the node. After creation, Kubernetes
-checks whether the node is valid or not. For example, if you try to create
-a node from the following content:
--->
-
-## 管理
-与 [Pods](/docs/concepts/workloads/pods/pod/) 和 [Services](/docs/concepts/services-networking/service/) 不同，
-节点并不是在 Kubernetes 中从头创建的：它们由外部的云服务商（例如 Google Compute Engine）创建，或者是来自你的资源池
-中的物理机或者虚拟机。
-这意味着当 Kubernetes 创建一个节点时，它其实仅仅创建了一个对象来代表这个节点。
-创建以后，Kubernetes 将检查这个节点是否可用。例如，如果你尝试使用如下内容创建一个节点：
-
-```json
-{
-  "kind": "Node",
-  "apiVersion": "v1",
-  "metadata": {
-    "name": "10.240.79.157",
-    "labels": {
-      "name": "my-first-k8s-node"
-    }
-  }
-}
-```
-
-<!--
-Kubernetes creates a node object internally (the representation), and
-validates the node by health checking based on the `metadata.name` field. If the node is valid -- that is, if all necessary
-services are running -- it is eligible to run a pod. Otherwise, it is
-ignored for any cluster activity until it becomes valid.
--->
-Kubernetes 会在内部创一个 Node 对象（用以表示节点），并基于 `metadata.name` 字段执行健康检查，
-如果节点合法，即所有必要服务都处于运行状态，它就有资格运行 Pod；否则它将被所有的集群活动忽略直到其变为合法为止。
-
-<!--
-{{< note >}}
-Kubernetes keeps the object for the invalid node and keeps checking to see whether it becomes valid.
-You must explicitly delete the Node object to stop this process.
-{{< /note >}}
---> 
-
-{{< note >}}
-Kubernetes 保留无效节点所对应的对象，并持续检查它是否合法。
-若要停止这种检查，必须显式删除 Node 对象。
-{{< /note >}}
-
-<!--
-Currently, there are three components that interact with the Kubernetes node
-interface: node controller, kubelet, and kubectl.
--->
-当前，有 3 个组件同 Kubernetes 节点接口交互：节点控制器、kubelet 和 kubectl。
+关于节点的一般性信息，例如内核版本、Kubernetes 版本（`kubelet` 和 `kube-proxy` 版本）、
+Docker 版本（如果使用了）和操作系统名称。这些信息由 `kubelet` 从节点上搜集而来。
 
 <!--
 ### Node Controller
 
-The node controller is a Kubernetes master component which manages various
-aspects of nodes.
+The node {{< glossary_tooltip text="controller" term_id="controller" >}} is a
+Kubernetes control plane component that manages various aspects of nodes.
 
 The node controller has multiple roles in a node's life. The first is assigning a
 CIDR block to the node when it is registered (if CIDR assignment is turned on).
 -->
-### 节点控制器
+### 节点控制器  {#node-controller}
 
-节点控制器是一个 Kubernetes 控制面组件，管理节点的方方面面。
+节点{{< glossary_tooltip text="控制器" term_id="controller" >}}是
+Kubernetes 控制面组件，管理节点的方方面面。
 
-节点控制器在节点的生命周期中扮演多个角色。第一个是当节点注册时为它分配一个 CIDR 区间（如果启用了 CIDR 分配）。
+节点控制器在节点的生命周期中扮演多个角色。
+第一个是当节点注册时为它分配一个 CIDR 区段（如果启用了 CIDR 分配）。
```

Comment:
Maybe:
```suggestion
第一个是当节点注册时为它分配一个 CIDR 区段（如果启用了 CIDR 区段分配）。
```
because the English original is using CIDR on the second occasion as a noun rather than an adjective, and this could be extra misleading for people encountering the foreign (ie, English) abbreviation without context.